src/test/groovy/life/qbic/utils/BioinformaticAnalysisSpec.groovy

```diff
@@ -91,7 +94,7 @@ class BioInformaticAnalysisSpec extends Specification {
         bioinformaticAnalysisParser.parseFrom(pathToDirectory)
         then:
         ParseException parseException = thrown(ParseException)
-        assert parseException.message.equals("Specified directory is empty")
+        assert parseException.message == ("Specified directory ${pathToDirectory.toString()} is empty")
```

Comment:
Shouldn't that be the DataParserException?

Reply:
You're right 👍